images/tensorflow/Dockerfile

```diff
@@ -21,9 +21,13 @@ RUN apt-get update && apt-get install -y --no-install-recommends git
 
 RUN pip3 install cloud-tpu-client pyyaml
 
+ARG tensorflow_text_version=latest
+ARG keras_version=latest
+
 # HACKs: Remove this once they are added to the Model Garden requirements
 RUN pip3 install tensorflow-recommenders --no-deps
-RUN pip3 install tensorflow-text==2.6.0rc0 --no-deps
+RUN pip3 install tensorflow-text==${tensorflow_text_version} --no-deps
```

Comment:
The `tensorflow-text==latest` syntax doesn't work:

```
root@822832b71ff2:/# pip install tensorflow-text==latest
ERROR: Could not find a version that satisfies the requirement tensorflow-text==latest
ERROR: No matching distribution found for tensorflow-text==latest
```

Reply:
Fixed it.